src/content/en/2019/a11y.md

```diff
@@ -130,21 +130,21 @@ A [main landmark](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARI
 
 We found **only one in every four pages** (26.03%) include a main landmark. And surprisingly, **8.06% of pages** erroneously contained more than one main landmark, leaving these users guessing which landmark contains the actual main content. So, make sure to double check you have exactly one!
 
-<insert graphic of metric 09_04>
+```<insert graphic of metric 09_04>```
 
 Alt text: Figure 4. Vertical bar chart titled "Number of “main" landmarks per page” displaying percentage data, ranging from 0 to 80 in increments of 20, vs. bars representing the number of “main” landmarks per page from 0 to 4. Source: HTTP Archive (July 2019). Zero: 73.97%; One: 17.97%; Two: 7.41%; Three: 0.15%; 4: 0.06%
 
 ### HTML section elements
 
 Since HTML5 was released in 2008, and was made the official standard in 2014, there are many HTML elements to aid computers and screen readers in understanding our page layout and structure.
 
-Elements like [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header), [`<footer>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer), [`<navigation>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav), and [<main>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) indicate where specific types of content live and allow users to quickly jump around your page. These are being used pretty widely across the web, with most of them being used on over **50% of pages** (`<main>` being the outlier).
+Elements like [`<header>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header), [`<footer>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer), [`<navigation>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav), and [`<main>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) indicate where specific types of content live and allow users to quickly jump around your page. These are being used pretty widely across the web, with most of them being used on over **50% of pages** (`<main>` being the outlier).
```

Comment:
This just fixes `[<main>]()` by wrapping main in backticks (because it's code). I hope that's ok?

Reply:
Yeah this chapter still needs a thorough editing pass and this can be ironed out then.